ANSWERS.md

```diff
@@ -1,7 +1,17 @@
 <!-- Answers to the Self Study Questions go here -->
 
 1. How would you teach the basic ideas behind preprocessing to a friend?  Please don't copy and paste an answer here, use your own thoughts.
+I'd explain that preprocessing is intended to simplify CSS and that everything we do with preprocessing should create reusable styles.
```

Comment:
It is also worth mentioning that Preprocessors extend CSS functionalities, allow for mixins, functions, variables etc.